docs/toolset/integration.md

```diff
@@ -95,11 +95,18 @@ myHost := payload.LocalEntity()
 
 cpu, err := myHost.NewMetricSet("CpuSample")
 cpu.SetMetric("cpuPercent", 75.0, metric.GAUGE)
+```
+
+RATE and DELTAs requires to belong to at least 1 attribute. As they are persisted on disk this attribute is used to "namespace" the metrics on the set so they don't collide with others with the same name.
```

Comment:
The `s` in `requires` is not necessary 
 
> RATE and DELTAs requires

I think that you can change the phrase to:

> RATEs and DELTAs should be grouped with at least 1 attribute

What do you think?

Reply:
I think the last sentence is misleading 🤔